dag-universities-a.py

```diff
@@ -1,12 +1,41 @@
+import os
 import logging
 from datetime import timedelta, datetime
+from shutil import ExecError
+from decouple import config
+import pandas as pd
+from sqlalchemy import create_engine
+from signal import pthread_kill
 from airflow import DAG
 from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import PythonOperator
 
 logging.basicConfig(level=logging.ERROR,
                     format='%(asctime)s - %(module)s - %(message)s',
                     datefmt='%Y-%m-%d')
 
+DIR = os.path.dirname(__file__)
+DB_USER = config('DB_USER')
+DB_PASSWORD = config('DB_PASSWORD')
+DB_HOST = config('DB_HOST')
+DB_NAME = config('DB_NAME')
+DB_PORT = config('DB_PORT')
+QUERY = f"{DIR}/sql/universidades-e.sql"
+
+def query():
+    try:
+        engine = create_engine(f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}".format(),
+                               echo=False, 
+                               client_encoding='utf8')
+        query = open(QUERY, 'r')
+        df_query = pd.read_sql_query(query.read(), engine)
+        if not os.path.exists(f'{DIR}/files'):
+            os.makedirs(f'{DIR}/files')

```

Comment:
La validación es necesaria, pero no es necesario hacerlo con un if. Te dejo un detalle sobre la función makedirs que podes aprovechar.

Syntax: os.makedirs(path, mode = 0o777, exist_ok = False)
Parameter:
path: A path-like object representing a file system path. A path-like object is either a string or bytes object representing a path.
mode (optional) : A Integer value representing mode of the newly created directory..If this parameter is omitted then the default value Oo777 is used.
exist_ok (optional) : A default value False is used for this parameter. If the target directory already exists an OSError is raised if its value is False otherwise not. For value True leaves directory unaltered.
Return Type: This method does not return any value.